util/chplenv/chpl_gpu.py

```diff
@@ -12,10 +19,27 @@
 #   Default GPU architecure for the vendor
 #   LLVM target
 GPU_TYPES = {
-    "cuda": ("CHPL_CUDA_PATH", "nvcc",  2,"sm_60",  "NVPTX"),
-    "rocm": ("CHPL_ROCM_PATH", "hipcc", 3,"gfx906", "AMDGPU")
+    "cuda": ("CHPL_CUDA_PATH", "nvcc",  2,"sm_60",  "NVPTX",
+             _validate_cuda_version),
+    "rocm": ("CHPL_ROCM_PATH", "hipcc", 3,"gfx906", "AMDGPU",
+             _validate_rocm_version)
 }
 
+def _reportMissingGpuReq(msg, allowExempt=True, suggestNone=True):
+    if suggestNone:
+        msg += " You can use the 'none' GPU layer by setting "
+        msg += "'CHPL_GPU_CODEGEN=none'."
+
```

Comment:
I'm not sure saying that it's "skipping generating GPU code" is right either.

I suppose you're right it is skipping generating GPU code in the sense that we don't run it through the LLVM backend, but we're still passing it through GpuTransforms and `here.gpus.size != 0`. `assertOnGpu()` will still fail if put in a non-GPUizable loop (I think?)

How about:

To avoid this issue, you can have GPU code run on the CPU by setting 'CHPL_GPU_CODEGEN=none'.

Reply:
I like that!